config/initializers/smtp.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+ActionMailer::Base.smtp_settings = {
+  domain: 'jv-training.herokuapp.com',
+  address: 'smtp.sendgrid.net',
+  port: 587,
+  authentication: :plain,
+  user_name: 'apikey',
+  password: ENV['SENDGRID_API_KEY']
```

Comment:
This env variable should be on the application.rb with the fetch method to check it